### PR TITLE
[do not review] Preview Rapidoc theme

### PR DESCRIPTION
**Pull request by kbatuigas**
Display cloud API reference using Rapidoc: [deploy preview](https://deploy-preview-297--redpanda-docs-preview.netlify.app/api/cloud-api.html)

### CODE REVIEW

**Comment by Deflaimun** [comment]
Was expecting to see coloured badges on the left nav. (GET green, POST blue, DELETE red) stuff like that. 

The small arrows to collapse/expand are weird. Can we change to a single button that changes the text? "Collapse/Expand" would be nice.

https://deploy-preview-297--redpanda-docs-preview.netlify.app/api/cloud-api.html#servers
https://deploy-preview-297--redpanda-docs-preview.netlify.app/api/cloud-api.html#auth

Those links don't have any description. They should have at least a very basic one.

**Comment by Deflaimun** [comment]
image
If I try to run an invalid function, it shows the response status as failed, but it opens the response box with the hourglass. Expectation was that the response box don't open. 

also, can we change `response status` style to red whenever something wrong happens?

**Comment by kbatuigas** [comment]
@Deflaimun sorry I missed your comments! Thanks so much for adding, we're also tracking suggestions in [this issue](https://github.com/redpanda-data/documentation-private/issues/2285) - let us know if you have more feedback!

**Comment by kbatuigas** [comment]
> Was expecting to see coloured badges on the left nav. (GET green, POST blue, DELETE red) stuff like that.
> 
> The small arrows to collapse/expand are weird. Can we change to a single button that changes the text? "Collapse/Expand" would be nice.
> 
> https://deploy-preview-297--redpanda-docs-preview.netlify.app/api/cloud-api.html#servers https://deploy-preview-297--redpanda-docs-preview.netlify.app/api/cloud-api.html#auth
> 
> Those links don't have any description. They should have at least a very basic one.

Yes, agreed with the servers and auth links. I will work on adding those - they can be added at least via the spec itself I believe

File: modules/ROOT/attachments/cloud-api.yaml [hunk 1]
```diff
@@ -0,0 +1,4351 @@
...
+          type: string
+        state:
+          $ref: '#/components/schemas/Operation.State'
+      readOnly: true
+      title: Operation
+      type: object
+  securitySchemes:
+    oauth:
+      description: |-
+        Request an API key for authenticating to Redpanda Cloud.</br>When you click <b>Get Token</b>, you'll be asked to log into your Redpanda Cloud account to authenticate.</br>After logging in, you'll be returned to this page. Then, whenever you try an endpoint from these docs, the API key will be applied automatically.
```
Comment (JakeSCahill) [inline modules/ROOT/attachments/cloud-api.yaml]:
```suggestion
        Request an API key for authenticating to Redpanda Cloud.</br>When you click <b>Get Token</b>, you'll be asked to log into your Redpanda Cloud account to authenticate.</br>After logging in, you'll be returned to this page. Then, whenever you try an endpoint from these docs, the API key is applied automatically.</br>The API key is valid for one hour.
```

**Comment by JakeSCahill** [comment]
@kbatuigas I think we can close this one.

**Comment by kbatuigas** [comment]
Closing as this is only to preview the Rapidoc theme, which has been implemented. Adding the cloud API reference is handled in separate PRs

**Pull request closed without merging**